src/pages/docs/getting-started/first-deployment/index.mdx

```diff
@@ -1,120 +1,169 @@
+2. In the **Featured** category, locate the Run a Script card and click **Add Step**.
 
 :::figure
-![Project questionnaire](/docs/getting-started/first-deployment/images/img-questionnairedialog.png "width=500")
+![Featured step templates](/docs/getting-started/first-deployment/images/add-step.png)
 :::
 
-## Create deployment process
+### Step Name
+You can leave this as the default _Run a Script_.
+
+### Script Source
+
+You can source script files via 3 methods:
+
+* Inline script (default)
+* Git repository
+* Package
 
-The next step in the journey is to create your deployment process. This is where you define the steps that Octopus uses to deploy your software. For our simple Hello World deployment, we will configure one step to print _Hello World_.
+3. Select **Inline script** as your script source. 
 
 :::figure
-![Create deployment process](/docs/getting-started/first-deployment/images/img-createdeploymentprocess.png "width=500")
+![Script source expander where users can select where to source scripts from](/docs/getting-started/first-deployment/images/script-source.png)
 :::
 
-1. Click **CREATE PROCESS** to see the available deployment steps.
-2. Select the **Script** tile to filter the types of steps.
-3. Scroll down and click **ADD** on the **Run a Script** tile.
-4. Accept the default name for the script.
-5. In the Execution Location section, select **Run once on a worker** if you're using Octopus Cloud, or **Run once on the Octopus Server** if you're using a self-hosted Octopus instance.
-6. Scroll down to the Script section, select your script language of choice, and enter the following script in the Inline Source Code section. 
+### Inline Source Code
+
+4. Select an appropriate script language.
+5. **Copy** the script below and **paste** it into the source code editor.
+
 ```
 Write-Host "Hello, World!"
 ```
 
-:::div{.hint}
-If you are using Octopus Cloud, Bash scripts require you to select the Hosted Ubuntu worker pool. The Default Worker Pool is running Windows and doesn't have Bash installed.
+:::figure
+![Inline source code expander where users can type a script](/docs/getting-started/first-deployment/images/inline-source-code.png)
 :::
 
-7. Click **SAVE** to save the step.
+### Execution Location
 
-## Release and deploy
+6. If you’re using Octopus Cloud, select **Run once on a worker**. 
+7. If you’re using a self-hosted Octopus instance, select **Run once on the Octopus Server**.
+
+:::figure
+![Execution location expander where users can select where this step will run](/docs/getting-started/first-deployment/images/execution-location.png)
+:::
 
-Next, we will create a release that we can deploy to our environments. A release is a snapshot of the deployment process and the associated assets (packages, scripts, variables) as they exist when the release is created.
+### Worker Pool
+
+8. Keep the default **Runs on a worker from a specific worker pool** option selected.
```

Comment:
Note that if the user is on a self-hosted instance, they won't see options for the worker pool because you've directed them to run the step on the Octopus Server in the previous step ☝️

Reply:
I've altered the content to be Octopus Cloud specific in [this commit](https://github.com/OctopusDeploy/docs/pull/2560/commits/72204744cfec023e2484e20cf3e90c75130a90ff)